shell/scripts/extension/helm/charts/ui-plugin-server/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 annotations:
   catalog.cattle.io/certified: rancher # Any application we are adding as a helm chart
-  catalog.cattle.io/kube-version: '>= 1.16.0-0 < 1.29.0-0'
+  catalog.cattle.io/kube-version: '>= 1.16.0-0 < 1.33.0-0'
```

Comment:
What's the reasoning behind using version `1.33`? The latest version of k8s is only `1.31`.

Reply:
Ah, I don't want to keep doing this all the time so I just gave it a shot 😛

Reply:
Fair enough. Perhaps we should get rid of this annotation all together? Depending on the extension someone is making would determine the compatible Kube version, I don't know how relevant it is to simply load an extension.